t/05-scheduler-full.t

```diff
@@ -213,7 +213,7 @@ subtest 're-scheduling and incompletion of jobs when worker rejects jobs or goes
     note 'waiting for job to be incompleted';
     for (0 .. 100) {
         last if $jobs->find(99982)->state eq OpenQA::Jobs::Constants::DONE;
-        sleep .2;
+        sleep .3;
```

Comment:
this is not "20" or "30" seconds but 0.3s, i.e. 300ms. And I don't see how this should fix any instabilities as we run this in a for loop. As this is about a sporadic issue – flaky test as the ticket says – please first get the fail ratio before and then after your fix, not just one change stating "it works"

Reply:
well, if you multiply by 100 (or actually 101), the number of loop iterations, you land at around 20 / 30 seconds.

Reply:
Yes, but if that is desired we can also increase the loop range

Reply:
I was only commenting on the `this is not "20" or "30" seconds`, which is wrong.